app/controllers/foreign/ForeignCountriesCheckYourAnswersController.scala

```diff
@@ -51,6 +52,7 @@ class ForeignCountriesCheckYourAnswersController @Inject() (
         rows = Seq(
           TotalIncomeSummary.row(taxYear, request.userAnswers),
           CountriesRentedPropertySummary.rowList(taxYear, request.userAnswers),
+          PropertyIncomeReportSummary.row(taxYear, request.user.isAgentMessageKey, request.userAnswers),
```

Comment:
The order of the CYA answers is slightly different from what's in the prototype ('Do you want to report...' comes before the list of countries), and each country is displayed horizontally instead of vertically